build/OutputDirectories.props

```diff
@@ -12,8 +12,7 @@
       <BaseIntermediateOutputPath>$([System.IO.Path]::GetFullPath('$(BaseOutputDirectory)/obj/$(MSBuildProjectName)'))</BaseIntermediateOutputPath>
       <OutputDirectory>$(BaseOutputDirectory)/dotnet</OutputDirectory>
       <Stage2WithBackwardsCompatibleRuntimesOutputDirectory>$(BaseOutputDirectory)/dotnetWithBackwardsCompatibleRuntimes</Stage2WithBackwardsCompatibleRuntimesOutputDirectory>
-      <!-- licavalc: we will need to change this depending on how we expose toolset version in the SDK. Right now, keeping the toolset version -->
-      <SdkOutputDirectory>$(OutputDirectory)/sdk/$(ToolsetVersion)</SdkOutputDirectory>
+      <SdkOutputDirectory>$(OutputDirectory)/sdk/$(SdkVersion)</SdkOutputDirectory>
```

Comment:
I think this is the better way for now while we have no uses of the toolset other than as a transport into core-sdk. Can you file a bug to think about this further?